src/speed-index.js

```diff
@@ -5,7 +5,7 @@ const imageSSIM = require('image-ssim');
 /* BEGIN FAST MODE CONSTANTS - See function doc for explanation */
 const fastModeAllowableChangeMax = 5;
 const fastModeAllowableChangeMedian = 3;
-const fastModeAllowableChangeMin = 1;
```

Comment:
drive-by fix since I thought this is what we decided already :)

doesn't affect any of the tests but prevents skipping very large gaps